source/FreeRTOS_Routing.c

```diff
@@ -442,22 +442,7 @@ struct xIPv6_Couple
  */
         NetworkEndPoint_t * FreeRTOS_FindEndPointOnIP_IPv6( const IPv6_Address_t * pxIPAddress )
         {
-            NetworkEndPoint_t * pxEndPoint = pxNetworkEndPoints;
-
-            while( pxEndPoint != NULL )
-            {
-                if( pxEndPoint->bits.bIPv6 != pdFALSE_UNSIGNED )
-                {
-                    if( xCompareIPv6_Address( &( pxEndPoint->ipv6_settings.xIPAddress ), pxIPAddress, pxEndPoint->ipv6_settings.uxPrefixLength ) == 0 )
-                    {
-                        break;
-                    }
-                }
-
-                pxEndPoint = pxEndPoint->pxNext;
-            }
-
-            return pxEndPoint;
+            return FreeRTOS_InterfaceEndPointOnNetMask_IPv6( NULL, pxIPAddress, 1 );
```

Comment:
Are we mixing FreeRTOS_FindEndPointOnIP & FreeRTOS_InterfaceEndPointOnNetMask?, Shoudn't it be FreeRTOS_FindEndPointOnNetMask_IPv6